script/Arcueid/Day 11/11_00_ARC11_2B.txt

```diff
@@ -73,7 +73,7 @@ After having dinner with Akiha, I leave the dining room.
 -- 今夜はわりと兄らしい振る舞いができたと思う。秋葉とも、琥珀さんとも何ら飾ることなく話が弾んだ。……や、今までに比べたら、の話だけど。
 // Interpreting 何ら飾ることなく話 as the opposite of うわべを飾る
 I think I was able to play the part of an older brother well enough.
- The conversation between Akiha, Kohaku-san and I was both down-to-earth and spirited.
+ The conversation between Akiha, Kohaku-san, and I was both down-to-earth and spirited.
```

Comment:
Submitter comment
>The conversation between akiha, kohaku-san, and *me*, not I
the sentence needs to still make sense if the other subjects are removed
"The conversation with I was down-to-earth and spirited" doesn't work

Reply:
Yeah I think that's correct